include/zephyr/net/net_time.h

```diff
@@ -50,61 +43,14 @@ extern "C" {
- *
- * Please use specific cycle/tick counter values rather than net_time_t whenever
- * possible especially when referring to the kernel system clock or values of
- * any single counter peripheral.
- *
- * net_time_t cannot represent general clocks referred to an arbitrary epoch as
- * it only covers roughly +/- ~290 years. It also cannot be used to represent
- * time according to a more complex timescale (e.g. including leap seconds, time
- * adjustments, complex calendars or time zones). In these cases you may use
- * @ref timespec (C11, POSIX.1-2001), @ref timeval (POSIX.1-2001) or broken down
- * time as in @ref tm (C90). The advantage of net_time_t over these structured
- * time representations is lower memory footprint, faster and simpler scalar
- * arithmetics and easier conversion from/to low-level hardware counter values.
- * Also net_time_t can be used in the network stack as well as in applications
- * while POSIX concepts cannot. Converting net_time_t from/to structured time
- * representations is possible in a limited way but - except for @ref timespec -
- * requires concepts that must be implemented by higher-level APIs. Utility
- * functions converting from/to @ref timespec will be provided as part of the
- * net_time_t API as and when needed.
- *
- * If you want to represent more coarse grained scalar time in network
- * applications, use @ref time_t (C99, POSIX.1-2001) which is specified to
- * represent seconds or @ref suseconds_t (POSIX.1-2001) for microsecond
- * resolution. Kernel @ref k_ticks_t and cycles (both specific to Zephyr) have
- * an unspecified resolution but are useful to represent kernel timer values and
- * implement high resolution spinning.
- *
- * If you need even finer grained time resolution, you may want to look at
- * (g)PTP concepts, see @ref net_ptp_extended_time.
- *
- * The reason why we don't use int64_t directly to represent scalar nanosecond
- * resolution times in the network stack is that it has been shown in the past
- * that fields using generic type will often not be used correctly (e.g. with
- * the wrong resolution or to represent underspecified concepts of time with
- * unclear syntonization semantics).
- *
- * Any API that exposes or consumes net_time_t values SHALL ensure that it
- * maintains the specified contract including all protocol specific tolerances
- * and therefore clients can rely on common semantics of this type. This makes
- * times coming from different hardware peripherals and even from different
- * network nodes comparable within well-defined limits and therefore net_time_t
- * is the ideal intermediate building block for timed network protocols.
  */
-typedef int64_t net_time_t;
+typedef uint64_t net_time_t;
 
 /** The largest positive time value that can be represented by net_time_t */
-#define NET_TIME_MAX INT64_MAX
+#define NET_TIME_MAX UINT64_MAX
 
 /** The smallest negative time value that can be represented by net_time_t */
```

Comment:
The comment is not valid after this change

Reply:
fixed.

Reply:
fixed.